tests.sh

```diff
@@ -84,21 +95,26 @@ prepare(){
     sed -i "/^access_points/ c\access_points: ['$ACCESS_POINT:$ACCESS_PORT']" $DAOS_AGENT_YAML
     sed -i "s/^\- .*/\- $ACCESS_POINT:$ACCESS_PORT/" $DAOS_CONTROL_YAML
 
+    if [ -z $NUM_TARGETS ] ; then
+        sed -i "s/targets:.*/targets: $NUM_TARGETS/" $DAOS_SERVER_YAML
+    fi
+
     #Create the daos_agent folder
     srun -n $SLURM_JOB_NUM_NODES mkdir  /tmp/daos_agent
     srun -n $SLURM_JOB_NUM_NODES mkdir  /tmp/daos_server
-}
 
-#Prepare log folders for tests
-prepare_test_log_dir(){
-    $SRUN_CMD create_log_dir.sh $1
+    # check arp settings
+    clush  --hostfile $RESULT_DIR/daos_all_hostlist "cat /proc/sys/net/ipv4/neigh/default/gc_thresh1 ;
+    cat  /proc/sys/net/ipv4/neigh/default/gc_thresh2 ;
+    cat  /proc/sys/net/ipv4/neigh/default/gc_thresh3 " | sort >> Log/$SLURM_JOB_ID/gc_thresholds
+
 }
 
 #Start DAOS agent
 start_agent(){
     echo -e "\nCMD: Starting agent...\n"
-    cmd="clush --hostfile Log/$SLURM_JOB_ID/daos_all_hostlist
-    -f $SLURM_JOB_NUM_NODES \"
+    fanout=$(($DAOS_CLIENTS+$DAOS_SERVERS+1))
+    cmd="clush -f $fanout --hostfile $RESULT_DIR/all_hostlist \"
```

Comment:
Can you explain the change here? Why not use fanout of $SLURM_JOB_NUM_NODES? Why not use daos_all_hostlist?